pytest/frontend/test_shared_ops.py

```diff
@@ -0,0 +1,211 @@
+# Copyright 2019 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+r"""Unit tests for the Strawberry Fields shared_ops module"""

```

Comment:
Good question. My thinking is no, since this is technically provided by the frontend UI package, and none of these tests actually require a backend to be loaded.

Reply:
how is it frontend?